quinn-proto/src/congestion/bbr/bw_estimation.rs

```diff
@@ -26,7 +26,7 @@ impl Default for BandwidthEstimation {
             prev_acked_time: None,
             total_sent: 0,
             prev_total_sent: 0,
-            sent_time: None,
+            sent_time: Instant::now(),
```

Comment:
Maybe comment to indicate this is a dummy value, rather than something semantically meaningful?

Reply:
Done.